src/libs/PolicyUtils.js

```diff
@@ -10,7 +10,13 @@ import ONYXKEYS from '../ONYXKEYS';
  * @returns {Array}
  */
 function getActivePolicies(policies) {
-    return _.filter(policies, (policy) => policy && policy.type === CONST.POLICY.TYPE.FREE && policy.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE);
+    return _.filter(
+        policies,
+        (policy) =>
+            policy &&
+            (policy.type === CONST.POLICY.TYPE.FREE || (policy.type === CONST.POLICY.TYPE.CORPORATE && policy.isPolicyExpenseChatEnabled)) &&
```

Comment:
This could be just 
```suggestion
            policy.isPolicyExpenseChatEnabled &&
```

If you apply this suggestion: https://github.com/Expensify/Web-Expensify/pull/38379/files#r1287557099